### PR TITLE
add endgames positions

### PR DESCRIPTION
**Pull request by sepiropht**

### CODE REVIEW

File: www/data/endgames.json [hunk 1]
```diff
@@ -0,0 +1,657 @@
...
+    "fen": "7k/R7/7K/8/8/1b6/8/8 w - - 0 1",
+    "name": "Basic ending XVIII"
+}, {
+    "eco": "18",
+    "fen": "6k1/5R2/6K1/8/8/8/8/6b1 w - - 0 1",
+    "name": "Basic ending XX"
+}, {
+    "eco": "19",
+    "fen": "8/8/8/8/8/3k4/r7/3NK3 w - - 0 1",
+    "name": "Basic ending XX1"
```
Comment (veloce) [inline www/data/endgames.json]: small typo here

File: src/js/ui/editor/menu.js [hunk 2]
```diff
@@ -80,6 +80,24 @@ export function renderSelectColorPosition(ctrl) {
         )
       ])
     ]),
+    m('div.select_input', [
+      m('label', {
+        'for': 'select_editor_positions'
+      }, 'Endgames'),
+      m('select.positions', {
+        id: 'select_editor_positions',
```
Comment (veloce) [inline src/js/ui/editor/menu.js]: The `id` must be different that the other selector's one.

File: src/js/ui/editor/menu.js [hunk 3]
```diff
@@ -80,6 +80,24 @@ export function renderSelectColorPosition(ctrl) {
...
+        onchange: function(e) {
+          ctrl.loadNewFen(e.target.value);
+        }
+      }, [
+        optgroup('Set the board', [
```
Comment (veloce) [inline src/js/ui/editor/menu.js]: You should add the empty `-- Position --` item like in the opening selector

Comment (sepiropht) [inline src/js/ui/editor/menu.js]: All changes done ! 
I have a question, how do you launch the project in browser ? Last week is worked well, 
but now after 'npm run watch', i have to do '  python -m SimpleHTTPServer 8080' 
If not, anything happen in browser. 
Even after that, it work, but i still have s cors issue and i'm not able to make any request

Comment (sepiropht) [inline src/js/ui/editor/menu.js]: I said 'nothing happens' but I should say that I get this error in the console
'Uncaught SecurityError: Failed to construct 'Worker': Script at 'file:///home/sepiropht/src/lichobile/www/lib/socketWorker.js' cannot be accessed from origin 'null'.'

Comment (sepiropht) [inline src/js/ui/editor/menu.js]: Ok it working like before. I think that line ' chromium-browser --user-data-dir=$HOME/.chromium_dev_dir --disable-web-security
 ' should remains in the readme shouldn't it ?

Comment (veloce) [inline src/js/ui/editor/menu.js]: yes that's the way to disable security in chrome

**Comment by veloce** [comment]
Thanks!

**Comment by sepiropht** [comment]
Yes. I would like to do more. What else can I do? I'm looking for a feature or a bug, but nothing seems obvious

**Comment by veloce** [comment]
Contributions are appreciated! Current bugs are not obvious, indeed. Instead I can suggest you one of the following features: #474, #473, or maybe #394 (should be the easiest).
Let me know if one of those suits you, and if you need help. Note that I'm currently in the process of migrating code to typescript, so any consequent new PR should be written in typescript.

**Comment by sepiropht** [comment]
Ok thanks for the response. I will look at them

**Comment by sepiropht** [comment]
I will try the first feature(game search), if it is too difficult, I will take the last

**Pull request merged**